#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-fix-retcode.dpatch by Øyvind Møll <oyvind@moll.no>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix return code.  Taken from debian bug: #329467.

@DPATCH@

--- old/src/interface.c	14 May 2005 09:52:02 -0000	1.5
+++ new/src/interface.c	4 Oct 2005 09:44:24 -0000
@@ -120,7 +120,7 @@
 	} 
 	pthread_mutex_unlock(&lock);
 
-	return NSS_STATUS_SUCCESS;
+	return retval;
 }
 
 enum nss_status
